scripts/install.sh

```diff
@@ -1,6 +1,104 @@
-#!/usr/bin/bash
+#!/bin/sh
 
+# Export variables
 export CMAKE_GENERATOR="Ninja"
+
+# Environment variables
+CURRENT_USER=$(whoami)
+CURRENT_PWD=$(pwd)
+
+# Colours
+RED="\e[1;31m"
+GREEN="\e[1;32m"
+YELLOW="\e[1;33m"
+BLUE="\e[1;34m"
+PURPLE="\e[1;35m"
+CYAN="\e[1;36m"
+RESET_COLOUR="\e[0m"
+
+# Set home based on user or root
+if [ ${CURRENT_USER} == "root" ]; then
+	CURRENT_HOME="/root"
+else
+	CURRENT_HOME="/home/${CURRENT_USER}"
+fi
```

Comment:
As I said earlier, I would find it better if the script doesn't clone the repo to the home directory

Reply:
Sorry, completely forgot it.

Reply:
Fixed by recent commit.